lib/tasks/support/teaching-internship-providers.md.erb

```diff
@@ -12,7 +12,7 @@ backlink: /
 fullwidth: true
 navigation: 5.55
 navigation_title: Can I do a teaching internship?
-navigation_description: Earn from £300 per week and find out what it is really like in the classroom with a teaching internship.
+navigation_description: Earn from <%= v :teachinginternship_weekly %> per week and find out what it is really like in the classroom with a teaching internship.
```

Comment:
@MylesJarvis You've probably seen that this doesn't render properly on `/is-teaching-right-for-me`. Let's discuss before looking for a dev solution!